xarray/conventions.py

```diff
@@ -637,6 +665,13 @@ def maybe_encode_dtype(var, name=None):
                                   'any _FillValue to use for NaNs' % name,
                                   RuntimeWarning, stacklevel=3)
                 data = duck_array_ops.around(data)[...]
+                if '_Unsigned' in encoding:
```

Comment:
From the documentation I assembled as part of [the same issue in NetCDF4-python](https://github.com/Unidata/netcdf4-python/issues/656), it looks like `_Unsigned = "true" is present if the data are `_Unsigned`, with no mention of `_Unsigned = "false"`. I haven't seen an example of `False` in the wild, but, your suggestion is consistent with the [patch to NetCDF4-python](https://github.com/Unidata/netcdf4-python/pull/658/files) for this issue.

I made the change you recommended on the encode side.

On decode there is a corresponding check of the attribute, so I made a change there, too.